e2e/app/component/card/data-resource-card.ts

```diff
@@ -66,42 +66,37 @@ export default class DataResourceCard extends BaseCard {
     return resourceCards;
   }
 
-  async findNameCellLinkFromTable(opts: { name?: string }): Promise<ElementHandle> {
-    const { name } = opts;
+  async confirmDataTableExist(name): Promise<DataTable> {
```

Comment:
looks good!  I have a nitpick about the name though: how about confirmDataTableHasResults ?